File: snippets/notify/create-notification-number/create-notification-number.rb
require 'rubygems'
require 'twilio-ruby'

account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

notification = @client.notify
                      .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                      .notifications
                      .create(
                         body: 'Knok-Knok! This is your first Notify SMS',
                         to_binding: {
                           'binding_type' => 'sms',
                           'address' => '+1651000000000'
                       }.to_json,
                         identity: ['identity']
                       )

puts notification.sid

File: snippets/notify/create-notification-number/create-notification-number.py
from twilio.rest import Client
import json

account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Client(account_sid, auth_token)

notification = client.notify \
                     .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX') \
                     .notifications \
                     .create(
                          body='Knock-Knock! This is your first Notify SMS',
                          to_binding= [
                            json.dumps({'binding_type': 'sms','address': '+1651000000000'}),
                            json.dumps({'binding_type': 'sms','address': '+1651000000001'})
                          ]
                      )

print(notification.sid)
